commands/build.go

```diff
@@ -253,6 +253,9 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 			}
 			return runBuild(dockerCli, options)
 		},
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return nil, cobra.ShellCompDirectiveFilterDirs
+		},
```

Comment:
Looks like this one was missed from the `completion` package?

Reply:
As it just returns `cobra.ShellCompDirectiveFilterDirs` I don't think we need a specific func in completion pkg if this is what you mean?